### PR TITLE
Expand glob range on source_gen

### PR DESCRIPTION
**Pull request by jcollins-g**
I recommend publishing this to break circular dependency problems with glob.

### CODE REVIEW

File: source_gen/pubspec.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ dependencies:
   async: ^2.0.7
   build: ^1.4.0
   dart_style: ^1.0.0
-  glob: ^1.1.0
+  glob: ">=1.1.0 <=3.0.0"
```
Comment (jakemac53) [inline source_gen/pubspec.yaml:15]:
```suggestion
  glob: ">=1.1.0 <3.0.0"
```

**Comment by jakemac53** [comment]
These test failures look erroneous, cc @kevmoo . It looks like the tests are not resilient to any change in log output.

**Review by jakemac53 — APPROVED** [review]

**Comment by kevmoo** [comment]
> These test failures look erroneous, cc @kevmoo . It looks like the tests are not resilient to any change in log output.

Yup.

**Comment by jakemac53** [comment]
Ok I am going to merge, tests can be fixed up separately and would be broken on master right now as well afaict.

**Pull request merged**